third_party/terraform/data_sources/data_source_storage_bucket_object_content.go

```diff
@@ -0,0 +1,61 @@
+package google
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"google.golang.org/api/storage/v1"
+)
+
+func dataSourceGoogleStorageBucketObjectContent() *schema.Resource {
+
+	dsSchema := datasourceSchemaFromResourceSchema(resourceStorageBucketObject().Schema)
+
+	addRequiredFieldsToSchema(dsSchema, "bucket")
+	addRequiredFieldsToSchema(dsSchema, "name")
+	addOptionalFieldsToSchema(dsSchema, "content")
```

Comment:
Should this be an output only field?

Reply:
It has been taken care of in https://github.com/GoogleCloudPlatform/magic-modules/blob/16c462951f174f835bb60bf481e8ec29278a3fb1/third_party/terraform/utils/datasource_helpers.go#L71:6

Reply:
Computed is false here. Should be true ?